### ISSUE TITLE
T10.57. Чи бувають віртуальні поля? Чому?

### ISSUE THREAD

**Issue by yanaskyrda** [kzhereb/knu-is-ooop2019#57]

**Comment by starquell** [comment]
Це не має сенсу, адже функцію поліморфізма виконують віртуальні методи

**Comment by AntonHladkiy** [comment]
В цьому немає сенсу, функцію  поліморфізма вже виконують virtual методи. Немає поведінки яку можна інкапсулювати в  звичайному доступі до змінної.

**Comment by RostyslavMV** [comment]
Не бувають і вони не потрібні, адже тоді це могло б порушити інкапсуляцію.

**Comment by RostyslavMV** [comment]
Не бувають і вони не потрібні, адже тоді це могло б порушити інкапсуляцію.

**Comment by i1red** [comment]
Ні. virtual дозволяє перевизначити методи, щоб змінити їх поведінку, тоді як поля(як змінні) не можуть мати поведінки взагалі

**Comment by ilyakorenevskyi** [comment]
Це не потрібно, адже для поліморфізму virtual змінює поведінку методу, для полів це не потрібно

**Comment by Beam-of-Light** [comment]
Не бувають і в них немає необхідності, адже virtual використовується для поліморфізму.
